src/options/read_all.rs

```diff
@@ -0,0 +1,76 @@
+use crate::{Credentials, Origin, Position, ReadDirection};
+
+#[derive(Clone)]
+pub struct ReadAllOptions {
+    pub(crate) credentials: Option<Credentials>,
+    pub(crate) direction: ReadDirection,
```

Comment:
**SUGGESTION:** Could we rename `ReadDirection` to `Direction` as it's in other clients?

Reply:
I won't mind but it is not exposed publicly

Reply:
Correction, the type is exposed but the user can use it on the API.